### PR TITLE
Territorial council page

### PR DESCRIPTION
**Pull request by kevin-verschaeve**

### CODE REVIEW

File: templates/base.html.twig [hunk 1]
```diff
@@ -184,6 +184,22 @@
                     {{ algolia_api_key ? ('\''~algolia_api_key|e('js')~'\'')|raw : 'null' }},
                     '{{ algolia_blacklist|base64_encode }}'
                 );
+
+                findAll(document, '.js--tabs').forEach(function (tab) {
```
Comment (kevin-verschaeve) [inline templates/base.html.twig:188]: do we have anywhere on the site something to switch tabs ? (I only saw tabs, that are news pages, not on the same page). If not, should this code be added here or somewhere else ?

File: templates/territorial_council/_nav.html.twig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<nav class="tabs-nav b__nudge--bottom-larger">
+    <ul>
+        <li class="active"><a href="{{ path('app_territorial_council_members') }}" class="tabs-nav__item">Membres</a></li>
+        <li><a href="#" class="tabs-nav__item">Élections internes</a></li>
```
Comment (ottaviano) [inline templates/territorial_council/_nav.html.twig:4]: maybe remove this empty link?

File: templates/adherent/contact.html.twig [hunk 3]
```diff
@@ -18,6 +18,10 @@
                     <a href="{{ path('app_citizen_project_show', {'slug': from.slug}) }}" class="back-to-list icon--with-text text--summary">
                         {{ include('components/caret--left.html.twig') }}
                         Retour au projet citoyen</a>
+                {% elseif fromType == 'territorial_council' %}
+                    <a href="{{ path('app_territorial_council_members') }}" class="back-to-list icon--with-text text--summary">
+                        {{ include('components/caret--left.html.twig') }}
+                        Retour au conseil territorial</a>
```
Comment (ottaviano) [inline templates/adherent/contact.html.twig:24]: `... au Conseil ...`?

File: templates/base.html.twig (hunk 1 above)
Comment (ottaviano) [reply templates/base.html.twig]: I think we can remove this block

**Comment by kevin-verschaeve** [comment]
I've added tests and fixed review comments

**Pull request merged**